scenarios/detection/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -41,14 +41,17 @@
   {
```

Comment:
<p>[minor] can we run the black formatter on this cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Done. Looks a lot nicer now</p>